opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -237,6 +237,7 @@ std::unique_ptr<Matrix> blockJacobiAdjacency(const Grid& grid,
               converged_(false),
               matrix_()
         {
+            OPM_TIMEBLOCK(istlSolverEbos);
```

Comment:
I think it is clearer to use `ISTLSolverEbos_constructor`.

Reply:
I can change it, but I think it is better to make it simple until more people have used it and see it is a problem.